cleanupBuckets.js

```diff
@@ -0,0 +1,160 @@
+const http = require('http');
+const async = require('async');
+const AWS = require('aws-sdk');
+const { Logger } = require('werelogs');
+
+const log = new Logger('s3utils::emptyBucket');
+// configurable params
+const BUCKETS = process.argv[2] ? process.argv[2].split(',') : null;
+const ACCESS_KEY = process.env.ACCESS_KEY;
+const SECRET_KEY = process.env.SECRET_KEY;
+const ENDPOINT = process.env.endpoint;
```

Comment:
capitalized `endpoint`? following what is used in `stalled.js`

Reply:
good catch